test-case/check-ipc-flood.sh

```diff
@@ -34,13 +38,17 @@ func_lib_check_sudo
 dlogi "Check sof debug fs environment"
 [[ "$(sudo file $ipc_flood_dfs|grep 'No such file')" ]] && dlogw "${BASH_SOURCE[0]} need $ipc_flood_dfs to run the test case" && exit 2
 dlogi "Checking ipc flood test!"
-dlogc "sudo bash -c 'echo $lpc_loop_cnt > $ipc_flood_dfs'"
-sudo bash -c "'echo $lpc_loop_cnt > $ipc_flood_dfs'"
 
-sof-kernel-log-check.sh $KERNEL_LAST_LINE
-[[ $? -ne 0 ]] && dloge "Catch error in kernel log" && exit 1
+for i in $(seq 1 $loop_cnt)
+do
+    dlogc "sudo bash -c 'echo $lpc_loop_cnt > $ipc_flood_dfs'"
+    sudo bash -c "'echo $lpc_loop_cnt > $ipc_flood_dfs'"
+
+    sof-kernel-log-check.sh 0
+    [[ $? -ne 0 ]] && dloge "Catch error in dmesg" && exit 1
 
-dlogi "Dumping test logs!"
-dmesg | grep "IPC Flood count" -A 2
+    dlogi "Dumping test logs!"
+    dmesg | grep "IPC Flood count" -A 2
+done
 
```

Comment:
can you add this line ` sof-kernel-log-check.sh $KERNEL_LAST_LINE ` before exit

so your code will look like:
```
sof-kernel-log-check.sh $KERNEL_LAST_LINE
exit $?
```